.gitignore

```diff
@@ -118,3 +118,24 @@ env/
 
 ### Configuration files ###
 .env
+
+### Ignore my command line file
+DockerCommands.txt
+
+### Ignore Terraform files
+.terraform.lock.hcl
+.terraform/
+.terraform/providers/registry.terraform.io/hashicorp/azurerm/4.3.0/darwin_amd64/terraform-provider-azurerm_v4.3.0_x5
+*.tfstate
+*.tfstate.backup
+*terraform.*
+.terraform
+
+# Ignore minikube
+minikube-darwin-amd64*
+
+#Ignore my local deployment .yaml
+#deployment.yaml
+
+#ignore secret.yaml
+secrect.yaml
```

Comment:
There is a typo here Cristiano.  You've spelt Secret wrong and so it isn't ignoring the secret.yaml.  In fact it is storing your secrets file in GIT. It's a simple mistake but it now means there is a potential security breach as I can see your secret API keys.